blocks/editable/tinymce.js

```diff
@@ -40,14 +45,18 @@ export default class TinyMCE extends Component {
 			this.editorNode.className = classnames( nextProps.className, 'blocks-editable__tinymce' );
 		}
 
-		if ( ! isEqual( this.props.aria, nextProps.aria ) ) {
-			const before = keys( this.props.aria );
-			const after = keys( nextProps.aria );
-			const removed = difference( before, after );
-			const updated = filter( after, ( key ) => ! isEqual( this.props.aria[ key ], nextProps.aria[ key ] ) );
-			forEach( removed, ( key ) => this.editorNode.removeAttribute( 'aria-' + key ) );
-			forEach( updated, ( key ) => this.editorNode.setAttribute( 'aria-' + key, nextProps.aria[ key ] ) );
-		}
+		const prevAriaKeys = getAriaKeys( this.props );
+		const nextAriaKeys = getAriaKeys( nextProps );
+		const removedKeys = difference( prevAriaKeys, nextAriaKeys );
+		const updatedKeys = differenceWith( nextAriaKeys, prevAriaKeys,
+				( nextKey, prevKey ) =>
+					nextKey === prevKey &&
+					nextProps[ nextKey ] === this.props[ prevKey ] );
```

Comment:
Why did you calculate the updated keys using an O(n^2) algorithm instead of the O(n) filter that I used originally?

Reply:
I wanted to see what this method could look like if we stuck to a single collection interface (`difference`/`differenceWith`) and whether it made it any clearer, but you're quite right: it's wasteful and doesn't do much for the reader. Amended and pushed.